dftimewolf/lib/collectors/virustotal.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+"""Downloads several items for a VT file."""
+
+import os
+import tempfile
+import urllib.parse
+import zipfile
+
+from typing import List
+from typing import Optional
+
+import vt
+
+from dftimewolf.lib import module
+from dftimewolf.lib.containers import containers
+from dftimewolf.lib.modules import manager as modules_manager
+from dftimewolf.lib.state import DFTimewolfState
+
+
+class VTCollector(module.BaseModule):
+  """VirusTotal (VT) Collector.
+
+  Attributes:
+    hashes_list List[hashes_list]: List of hashes passed ot the module
+    vt_type: pcap or evtx depending on the file type requested
+
+  """
+
+  def __init__(
+      self,
+      state: DFTimewolfState,
+      name: Optional[str],
+      critical: bool = False) -> None:
+    """Initializes an VirusTotal (VT) collector.
+
+    Args:
+      state: recipe state.
+      name: The module's runtime name.
+      critical: True if the module is critical, which causes
+          the entire recipe to fail if the module encounters an error.
+    """
+    super(VTCollector, self).__init__(state, name=name, critical=critical)
+    self.hashes_list: List[str] = []
+    self.directory: Optional[str] = None
+    self.client: Optional[vt.Client] = None
+    self.vt_type = None
+
+  def Process(self) -> None:
+    """Process of the VirusTotal collector after setup"""
+
+    for vt_hash in list(self.hashes_list):
+      if not self._isHashKnownToVT(vt_hash):
+        self.logger.info(
+            f'Hash not found on VT removing element {vt_hash} from list')
+        self.hashes_list.remove(vt_hash)
+
+    self.logger.info(f'Found the following files on VT: {*self.hashes_list,}')
```

Comment:
Just doing 
```suggestion
    self.logger.info(f'Found the following files on VT: {str(self.hashes_list)}')
```

Might be cleaner